### DISCUSSION TITLE
Protect a page with BankID

### DISCUSSION THREAD

**Discussion by danceshorribly** [jamieblomerus/WP-Mobile-BankID-Integration#12] [Support] [answered]
Hi!
Great job! Awaiting live creds from my bank but I think this plugin will work out with some tweaks.

Trying to set it up so that the user first need to be authenticated via BankID in order to view the site.

The site will almost exclusively be used via mobile so the idea is to use a button to trigger the BankID-app to open and then redirect back to the site.

**Comment by jamieblomerus** [comment, marked as answer]
@danceshorribly Requiring users to log in with Mobile BankID sounds like a solid solution. It might involve some coding, but if you’re up for it, it’s worth exploring. 

Since there are already plugins available for requiring login before site access, integrating with one of them could streamline the process.

Good luck with setting it up! 🚀

**Comment by danceshorribly** [comment]
You think it would be possible to implement this to be used for password-protected pages?

**Reply by jamieblomerus** [reply]
Yeah, I think so. But it will probably require some custom development.
